README.md

```diff
@@ -118,3 +118,39 @@ in parallel for free:
 # Sample 4 chains from the posterior.
 chain = psample(model, RWMH(init_params), 100000, 4; param_names=["μ","σ"], chain_type=Chains)
 ```
+
+## Metropolis-adjusted Langevin algorithm (MALA)
+
+AdvancedMH.jl also offers an implementation of [MALA](https://en.wikipedia.org/wiki/Metropolis-adjusted_Langevin_algorithm) if the `ForwardDiff` and `DiffResults` packages are available. 
+
+A set of `init_params` are required. `x->` within the `MALA` struct takes the gradient computed at the current sample.
```

Comment:
Maybe it would make sense to implement `propose(rng::Random.AbstractRNG, ::MALA, model) = error("please specify initial parameters")` since otherwise users will only get a dispatch error.

I think it might also be better to explain the constructor instead of the struct (which should not matter for users):
```suggestion
A `MALA` sampler can be constructed by `MALA(proposal)` where `proposal` is a function that
takes the gradient computed at the current sample. It is required to specify an initial sample `init_params` when calling `sample`.
```

(BTW: It's good that it is the first argument since this allows one to write
```julia
sampler = MALA() do x
  # some complicated calculations
  ....
  return proposal
end
```)